sys/include/net/gnrc/sixlowpan/ctx.h

```diff
@@ -23,7 +23,7 @@
  *
  * @author  Martine Lenders <mlenders@inf.fu-berlin.de>
  */
-#ifndef GNRC_SIXLOWPAN_CTX_H_
+#if !(defined(GNRC_SIXLOWPAN_CTX_H_)) && defined(MODULE_GNRC_SIXLOWPAN_CTX)
```

Comment:
It seems to me you want to exclude the function definitions if the module isn't present, but that will be noticed by the linker anyway. Why is `net/gnrc/sixlowpan/ctx.h` different from other module headers? Will we introduce this for other modules, too?